src/messaging/ExchangeContext.cpp

```diff
@@ -239,14 +239,28 @@ CHIP_ERROR ExchangeContext::SendMessage(Protocols::Id protocolId, uint8_t msgTyp
             }
         }
 
-        // Standalone acks are not application-level message sends.
-        if (!isStandaloneAck && err == CHIP_NO_ERROR)
+        if (err == CHIP_NO_ERROR)
```

Comment:
Should this just be "else" on the preceding if, if we change the condition like this?